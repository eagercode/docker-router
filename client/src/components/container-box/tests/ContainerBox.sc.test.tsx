import * as React from 'react';
import { mount } from 'enzyme';
import 'jest-styled-components';
import Div from '../ContainerBox.sc';

describe('<Div/>', () => {
    it('container box styled div snapshot', () => {
        expect(mount(<Div/>)).toMatchSnapshot();
    });
});
