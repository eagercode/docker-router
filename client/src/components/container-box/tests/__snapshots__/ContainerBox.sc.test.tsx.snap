// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Div/> container box styled div snapshot 1`] = `
.c0 {
  box-shadow: 0 0 4px 2px #ccc;
  display: inline-block;
  margin: 10px 2px 10px 2px;
  position: relative;
  width: 100%;
  word-wrap: break-word;
}

<styled.div>
  <div
    className="c0"
  />
</styled.div>
`;
